### PR TITLE
fixed typo and style problems

### PR DESCRIPTION
**Pull request by a-m-thomas**

### CODE REVIEW

**Review by lennessyy — APPROVED** [review]

File: modules/tutorials/pages/pattern-matching/load-data.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ Below, we use the GSQL loading language to define a loading job script, which en
 
 The script to load the LDBC-SNB data is below.
 
-.# GSQL script for loading LDBC-SNB data
+.GSQL script for loading LDBC-SNB data
```
Comment (victor-gsl) [inline modules/tutorials/pages/pattern-matching/load-data.adoc:9]: The reason why this line began with # is because it is supposed to be a comment line in the code.  # was my favorite comment marker, but sadly it is deprecated.  Either
1. Put the line into the code block where it belongs, beginning with //
or 
2. Remove the line as unncessary.

File: modules/tutorials/pages/pattern-matching/load-data.adoc [hunk 2]
```diff
@@ -126,7 +126,7 @@ CREATE LOADING JOB load_ldbc_snb FOR GRAPH ldbc_snb {
 
 We have generated a data set with scale factor 1 (approximate 1GB). You can download it from https://s3-us-west-1.amazonaws.com/tigergraph-benchmark-dataset/LDBC/SF-1/ldbc_snb_data-sf1.tar.gz
 
-.# Linux Bash 
+.Linux Bash
```
Comment (victor-gsl) [inline modules/tutorials/pages/pattern-matching/load-data.adoc:129]: ditto

File: modules/tutorials/pages/pattern-matching/multiple-hop-and-accumulation.adoc [hunk 3]
```diff
@@ -153,7 +153,7 @@ WHERE x1.age>x2.age AND x2.@cnt!=x4.@cnt AND (x1.salary+x3.salary)<x4.salary
 GSQL's pattern matching syntax provides the essentials for a regular expression language for paths in graphs.  Consider the three basic requirements for a regular expression language:
 
 . *The empty set* --> A path of length zero (no match)
-. *Concatenation* --> Form a path by adding one on two another. You can write an N-hop pattern, and M-hop pattern, and then combine them to have a (N+M)-hop pattern.
+. *Concatenation* --> Combine paths together. You can write an N-hop pattern, and M-hop pattern, and then combine them to have an (N+M)-hop pattern.
```
Comment (victor-gsl) [inline modules/tutorials/pages/pattern-matching/multiple-hop-and-accumulation.adoc:156]: Concatenate is more precise than the combine: "Form a longer path by chaining one path after another one."

File: modules/tutorials/pages/pattern-matching/multiple-hop-and-accumulation.adoc [hunk 4]
```diff
@@ -566,13 +553,13 @@ INTERPRET QUERY () SYNTAX v2 {
 }
 ----
 
-*Example 2.* Find Viktor Akhiezer's liked posts' authors A, and his liked comments' authors B. Count the common universities that both A and B have members studied at.
+*Example 2.* Find Viktor Akhiezer's liked posts' authors A, and his liked comments' authors B. Count the universities that members from groups A and B studied at.
```
Comment (victor-gsl) [inline modules/tutorials/pages/pattern-matching/multiple-hop-and-accumulation.adoc:556]: I think the rewrite loses the sense that we only want universities that were attended by students from both A and B.  The word "both" is important.

File: modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc [hunk 5]
```diff
@@ -13,37 +13,43 @@ Currently, pattern matching may only be used in read-only queries. DML support w
...
-In classic GSQL queries, described in xref:tutorials:gsql-101/README.adoc[GSQL 101], we used the punctuation `+-( )->+`  in the FROM clause to indicate a 1-hop query, where the arrow specifies the vertex flow from left to right, and `( )` encloses the edge types.
+In classic GSQL queries, described in xref:tutorials:gsql-101/README.adoc[GSQL 101], we used the punctuation `+-( )->+`  in the FROM clause to indicate a 1-hop query.
 
+There the arrow `+->+` specifies the vertex flow from left to right and the parentheses `( )` enclose the edge types.
+
+.Classic GSQL Example
 [source,gsql]
 ----
-Person:p -(LIKES:e)-> Message:m          /* Classic GSQL example */
+Person:p -(LIKES:e)-> Message:m
```
Comment (victor-gsl) [inline modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc:30]: This is a good example of good and consistent use of empty space in a path pattern. In template form:
`V1:v1 -(E1:e1)- V2:v2 -(E2:e2)- V3:v3`
I think the dashes - should be directly adjacent to the parentheses: -(edge)-
I prefer to have a space separating the dash from the vertex. This space could be eliminated, but the same rule should be applied for the entire path. Here is an ugly example of bad, inconsistent spacing, on the multiple-hop-and-accumulation page:
`Person:s-(LIKES>) -:msg - (HAS_CREATOR>)-Person:t`
Can you check FROM clauses and improve the path pattern spacing?

File: modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc [hunk 6]
```diff
@@ -91,13 +97,13 @@ Performance may be better when types are explicitly provided.
  ** Any right directed edge with source in X and target in Y.
 . `FROM X:x - (<_:e) - Y:y`
  ** Any left directed edge with source in Y and target in X.
-. `FROM X:x - ((<_|_):e) - Y:y`
- ** Any left directed or any undirected; "|" means OR, and parentheses enclose the group of edge descriptors; e is the alias for the edge pattern (<_|_).
+. `+FROM X:x - ((<_/_):e) - Y:y+`
+ ** Any left directed or any undirected; `|` means OR, and parentheses enclose the group of edge descriptors; e is the alias for the edge pattern `+(<_/_)+`.
```
Comment (victor-gsl) [inline modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc:101]: Why did you change | to /?
Your new way is incorrect.

File: modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc [hunk 7]
```diff
@@ -91,13 +97,13 @@ Performance may be better when types are explicitly provided.
...
 . `FROM X:x - ((E1|E2>|<E3):e) - Y:y`
  ** Any one of the three edge patterns.
 . `FROM X:x - () - Y:y`
  ** any edge (directed or undirected)
- ** Same as `(<_|_>|_)`
+ ** Same as `+(<_/_>|_)+`
```
Comment (victor-gsl) [inline modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc:106]: ditto
